kafka-ui-react-app/src/components/Version/Version.tsx

```diff
@@ -35,13 +35,9 @@ const Version: React.FC = () => {
     <S.Wrapper>
       <S.CurrentVersion>{currentVersion}</S.CurrentVersion>
 
-      {!!outdated && (
-        <S.OutdatedWarning
-          title={`Your app version is outdated. Current latest version is ${latestTag}`}
-        >
-          <WarningIcon />
-        </S.OutdatedWarning>
-      )}
+      <S.OutdatedWarning title="You have disabled version check, can’t fetch actual version">
+        <WarningIcon />
+      </S.OutdatedWarning>
```

Comment:
What's going on here? It looks like we going to disable version check for all users?

Reply:
Roma told me to set it as a hard code for a while while the back end does its part.